doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+
+!media lfr/MCCgenProc.jpg
+       style=width:100%
+       id=MCCgenProc
+       caption=MC2-3 cross section generation procedure with the axial leakage effect considered using TWODANT
+
+Without `l_spatial_homogenization=F` in an input, which is easy to make mistake, one will end up using the assembly average cross section for each region in heterogeneous fine-mesh transport calculation. [XS_Hom_Vs_Reg] shows the relative difference of assembly average cross section to hexagonal ring-wise fuel cross section for U-238 capture. These differences are basically the same as the spatial self-shielding factor of each ring: the ratio of hexagonal ring-wise flux to assembly average flux. Since spectrum is softer for peripheral regions than for central regions, central region flux is lower than the assembly average flux at low energy and higher at high energy. Thus, a test showed that the use of assembly average cross section instead of region-wise cross section resulted in about +400 pcm error in k-effective due to underestimation of low energy capture and overestimation of high energy fission.
+
+!media lfr/XS_Hom_Vs_Reg.jpg
+       style=width:50%
+       id=XS_Hom_Vs_Reg
+       caption=Relative difference of assembly average cross section to hexagonal ring-wise fuel cross section for U-238 capture
+
+For the non-fuel region, the setting is almost similar to the first step input except `c_externalspectrum_ufg=rzmflx` instead of `l_twodant=T` in the `control` block and `i_externalspectrum(composition #)=TWODANT zone #` in the `material` block.
+
+!listing lfr/heterogeneous_single_assembly_3D/cross_section/Step2/NonFuel/LFR_127Pin_NonFuel_9g.mcc3.sh
+
+### Run scripts and merging ISOTXS files
+
+The process to prepare the final isoxml file is described here. First, the TWODANT running script is run as
+
+```
+cd Step1
+./LFR_InnerFuel_HOM_1041g_TWODANT.mcc3.sh
+```
+
+Once the `LFR_InnerFuel_HOM_1041g_TWODANT.mcc3.sh.rzmflx` file is created, two scripts need to be run as
+
+```
+cd ../Step2/Fuel
+./LFR_127Pin_Fuel_1D_9g.mcc3.sh
+cd ../NonFuel
+./LFR_127Pin_NonFuel_9g.mcc3.sh
+cd ../../
+```
+
+The order to run two scripts doesn't matter. Once the above two scripts are run, `LFR_127Pin_Fuel_1D_9g.mcc3.sh.ISOTXS` and `LFR_127Pin_NonFuel_9g.mcc3.sh.ISOTXS` files are created. The next step is to convert those two ISOTXS files to isoxml files by running
+
+```
+isoxml-opt -n ./Step2/Fuel/LFR_127Pin_Fuel_1D_9g.mcc3.sh.ISOTXS
```

Comment:
Maybe it is better to replace `isoxml-opt -n` with `griffin-opt --isoxml--nisotxs` now because it is a bit hard to explain what `isoxml-opt` is.

Reply:
Accepted.